tasks/main.yaml

```diff
@@ -35,8 +35,9 @@
 
 - name: Download Apache Kafka
   get_url:
-    url: "{{ kafka_download_base_url }}/{{ kafka_version }}/kafka_{{ kafka_scala_version }}-{{ kafka_version }}.tgz"
+    url: '{{ kafka_download_base_url }}/{{ kafka_version }}/kafka_{{ kafka_scala_version }}-{{ kafka_version }}.tgz'
     dest: /tmp
+    validate_certs: '{{ kafka_download_validate_certs }}'
```

Comment:
The Lets Encrypt certificate used by this server can cause cert expiration issues depending on the version of OpenSSL and cacerts that are installed on a machine. Setting this to `no` will workaround certificate problems when the task attempts to download the tar package, CentOS 7 is one such OS where this is an issue. cURL works fine but wget does not.